installer/IP12_avm_freetz_ng_extract.sh

```diff
@@ -81,17 +82,16 @@ IP12_avm_freetz_ng_extract() {
         if ! [[ -d external/freetz-ng ]]; then
           mkdir external/freetz-ng
 
-          chown -R freetzuser:freetzuser external/freetz-ng
-          chmod 777 -R external/freetz-ng
-          su freetzuser -c "git clone https://github.com/Freetz-NG/freetz-ng.git external/freetz-ng"
+          su - freetzuser -c "git clone https://github.com/Freetz-NG/freetz-ng.git /tmp/freetz-ng"
```

Comment:
as we are using  `"$EXT_DIR"/freetz-ng/fwmod in the extractor ` this will not work.

Reply:
https://github.com/e-m-b-a/emba/blob/992947c9e865d9dca232fd523fc7c5e6cca07393/modules/P12_avm_freetz_ng_extract.sh#L57